source/content/start-states/wordpress.md

```diff
@@ -14,6 +14,25 @@ For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-s
 
 ## Latest Release
 
+### 2022-07-12
+
+#### <a name="20220606-1" class="release-update"></a>Show a more meaningful notice when trying to install WP plugin in Git mode
+
+If a site is in Git mode, the plugin installation UI is disabled and a notice is added to the plugins page.
+
+#### <a name="20220603-1" class="release-update"></a>Remove call to Pantheon API from WP admin
+
+Most of the time the Pantheon API requests are needless (e.g. there is no update available) and beyond the negative effect that has on the system as a whole, it also can cause a poor experience for the WordPress user when the API call fails under load. Additionally, reduced reliance on the API means more reliability, and less complexity in terms of checking for error conditions and edge cases.
+
+The update notice behavior will be as follows:
+
+- If a WP Core update is detected (via get_core_updates), the update notice is shown on each admin page.
```

Comment:
```suggestion
- If a WP Core update is detected (via `get_core_updates`), the update notice is shown on each admin page.
```

Reply:
just for consistency - it doesn't definitely affect readability without the backticks, but because it's a line in code/a file, we generally code-syntax it

otherwise 🚀

Reply:
Thanks @EdwardAngert!